CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.24.1] - 2022-11-11
+
+## Fixed
+
+- Fixed issue with bulletin board Ruby client OpenSSL 3.0 compatibility
```

Comment:
This was manually added? Should we add the other changes too? 

https://github.com/decidim/decidim-bulletin-board/compare/v0.24.0...develop

Reply:
Yes, this is manually created as we don't have automation in this repo.

Aren't they mostly documentation / local configuration changes?

I think they are not relevant in the changelog, I would categorize them in the Decidim main repo as `internal`.

Also looking at the previous entries in the changelog, I don't think it includes everything.

Reply:
Ok, maybe we'd need to standardize the labels/changelog generation/etc to be consistent with `decidim/decidim`. But at the moment as this project isn't used in production I think that what we have is good then.